osu.Game/Screens/Select/BeatmapPlaylistItem.cs

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Events;
+using osu.Framework.Localisation;
+using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Online.Multiplayer;
+using osuTK;
+using osuTK.Graphics;
+using osuTK.Input;
+
+namespace osu.Game.Screens.Select
+{
+    public class BeatmapPlaylistItem : Container
+    {
+        private const int fade_duration = 60;
+        public readonly Bindable<PlaylistItem> PlaylistItem = new Bindable<PlaylistItem>();
+        private readonly UpdateableBeatmapBackgroundSprite cover;
+        private readonly DragHandle dragHandle;
+        private readonly RemoveButton removeButton;
+        private bool isHovered;
+        private bool isDragged;
+
+        public event Action<BeatmapPlaylistItem> RemovalTriggered;
+
+        private class DragHandle : SpriteIcon
+        {
+            public DragHandle()
+            {
+                Anchor = Anchor.CentreLeft;
+                Origin = Anchor.CentreLeft;
+                Size = new Vector2(12);
+                Icon = FontAwesome.fa_bars;
+                Alpha = 0;
+                Margin = new MarginPadding { Left = 5, Top = 2 };
+            }
+
+            public override bool HandlePositionalInput => IsPresent;
```

Comment:
sure you don't want some kind of hover fade animation? is this a temporary thing?